polytope/polytope.py

```diff
@@ -1292,25 +1301,26 @@ def volume(polyreg):
         N = 50
     elif n == 2:
         N = 500
-    elif n ==3:
+    elif n == 3:
         N = 3000
     else:
         N = 10000
 
     l_b, u_b = polyreg.bounding_box
-    x = np.tile(l_b,(1,N)) +\
-        np.random.rand(n,N) *\
-        np.tile(u_b-l_b,(1,N) )
+    x = np.tile(l_b, (1, N)) +\
```

Comment:
The second is better because it is visually cleaner. Also, in general, adding extra parenthesis implies a change from normal order of operation, but this is not the case here.